src/Model/Resolver/MoveWishlistToCart.php

```diff
@@ -95,29 +96,56 @@ public function __construct(
      */
     protected function addItemsToCart(array $wishlistItems, CartInterface $quote): void
     {
-        foreach ($wishlistItems as $item) {
-            $product = $item['product'];
+        $cartItems = $quote->getItems();
+        $addedQty = false;
```

Comment:
Naming `addedQty` implies having a quantity stored in that variable. Despite this, a boolean value is stored.